tests/resources/worker/steps/transformer/expected/UTDANNING_PATCH.json

```diff
@@ -0,0 +1 @@
+{"name": "UTDANNING", "temporality": "STATUS", "languageCode": "no", "sensitivityLevel": "PERSON_GENERAL", "populationDescription": "Personer med utdannelse i Norge", "subjectFields": ["BEFOLKNING", "SAMFUNN"], "temporalCoverage": {"start": null}, "measureVariable": {"name": "UTDANNING", "label": "Utdanning", "notPseudonym": true, "dataType": "String", "variableRole": "Measure", "representedVariables": [{"description": "Utdanningsniv\u00e5 for person", "validPeriod": {"start": -16071}, "valueDomain": {"codeList": [{"category": "Grunnskole", "code": "1"}, {"category": "Videreg\u00e5ende skole", "code": "2"}, {"category": "Bachelorgrad", "code": "3"}, {"category": "Mastergrad", "code": "4"}, {"category": "Doktorgrad", "code": "5"}], "missingValues": []}}]}, "identifierVariables": [{"name": "PERSONID_1", "label": "Personidentifikator", "notPseudonym": false, "dataType": "Long", "variableRole": "Identifier", "representedVariables": [{"description": "Identifikator for person i microdata", "validPeriod": {"start": null}, "valueDomain": {"description": "Pseudonymisert personnummer", "unitOfMeasure": "N/A"}}], "format": "RandomUInt48", "keyType": {"name": "PERSON", "label": "Person", "description": "Person er et enkeltmenneske, individ."}}], "attributeVariables": [{"name": "START", "label": "Startdato", "notPseudonym": true, "dataType": "Instant", "variableRole": "Start", "representedVariables": [{"description": "Startdato/m\u00e5letidspunktet for hendelsen", "validPeriod": {"start": null}, "valueDomain": {"description": "Dato oppgitt i dager siden 1970-01-01", "unitOfMeasure": "N/A"}}]}, {"name": "STOP", "label": "Stoppdato", "notPseudonym": true, "dataType": "Instant", "variableRole": "Stop", "representedVariables": [{"description": "Startdato/m\u00e5letidspunktet for hendelsen", "validPeriod": {"start": null}, "valueDomain": {"description": "Dato oppgitt i dager siden 1970-01-01", "unitOfMeasure": "N/A"}}]}]}
```

Comment:
I don't see the temporalStatusDates field in the expected file. Shouldn't the test expose the bug?

Reply:
The issue was that an exception was thrown on the non existing key 'temporalStatusDates'. Since PATCH_METADATA is a valid case and would not produce this field it should not raise this exception, and carry on.

In a PATCH_METADATA the temporalStatusDates field is not allowed to be overwritten, and as such is not touched by the following processing:

in the patch method:
```py
"temporalStatusDates": self.temporal_status_dates
```
(Where self=the released metadata, and other=the patched metadata

The temporalStatusDates is not serialized to Null but ignored if not existing:
```py
if self.temporal_status_dates is None:
    del metadata_dict['temporalStatusDates']
return metadata_dict
```
(It will always be None in the model as it is an optional field, but is not relevant in the json representation if Null)

Reply:
That explains it!

Reply:
🙇🏻 
I will run a full test in admin with a STATUS dataset when the changes are deployed as well 👍🏻